samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/prj.conf

```diff
@@ -80,3 +80,41 @@ CONFIG_BT_MESH_LABEL_COUNT=3
 #CONFIG_BT_MESH_SELF_TEST=y
 
 CONFIG_GPIO=y
+
+#### SMP_SVR Related Stuff (START) ####
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Ensure an MCUboot-compatible binary is generated.
```

Comment:
It seems like that is the opposite of what setting this to `n` accomplishes. What is the goal here?